search_params.py

```diff
@@ -125,13 +154,18 @@ def main():
     parser = argparse.ArgumentParser()
     parser.add_argument(
         '--config', help='Path to configuration file (default: %(default)s). Please specify a config with all arguments in LibMultiLabel/main.py::get_config.')
-    parser.add_argument('--cpu_count', type=int, default=4, help='Number of CPU per trial (default: %(default)s)')
-    parser.add_argument('--gpu_count', type=int, default=1, help='Number of GPU per trial (default: %(default)s)')
-    parser.add_argument('--local_dir', default=os.getcwd(), help='Directory to save training results of tune (default: %(default)s)')
+    parser.add_argument('--cpu_count', type=int, default=4,
```

Comment:
As discussed, should we remove the new lines?

Reply:
Could we unify to 
```
    parser.add_argument('--vocab_file', type=str,
                        help='Path to a file holding vocabuaries (default: %(default)s)')
```
Making help on second line, which could be easier to read than single line?

Reply:
Maybe we can follow the rules (see section indentation) here: https://www.python.org/dev/peps/pep-0008

Reply:
Yes. But there are many different styles for [indentation](https://www.python.org/dev/peps/pep-0008/#indentation)
For the args we use this style for now.
```
# Aligned with opening delimiter.
foo = long_function_name(var_one, var_two,
                         var_three, var_four)
```

Let's discuss if we want to unify the style